packages/core/src/pagination/register.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@cds/core/button/register.js';
+import '@cds/core/icon/register.js';
+import { registerElementSafely } from '@cds/core/internal';
+import { CdsPaginationButton } from './pagination-button.element.js';
+import { CdsPaginationNumber } from './pagination-number.element.js';
+import { CdsPagination } from './pagination.element.js';
+
+registerElementSafely('cds-pagination', CdsPagination);
+registerElementSafely('cds-pagination-number', CdsPaginationNumber);
+registerElementSafely('cds-pagination-button', CdsPaginationButton);
+
```

Comment:
We need to do `ClarityIcons.addIcons()` for all the icons we are using in the pagination button. That's usually done in this file.